### DISCUSSION TITLE
Sponsors FAQs (Frequently Asked Questions) :octocat:

### DISCUSSION THREAD

**Discussion by DHayes10** [community/community#116811] [Other Feature Feedback, Questions, & Ideas]
As the **Sponsors program grows and becomes available in more regions**, we've **listened to your feedback and heard you loud and clear.** We are thrilled to provide **answers to some of your most asked questions in one place**. Also, making it easier to **search the docs with quick links for any additional information.** 

If you don’t see your question, need a little extra guidance, or want to share any feedback please feel free to [post it in the community](https://github.com/orgs/community/discussions/new?category=sponsors). We'll be keeping this FAQ up-to-date with the latest changes and updates that will be helpful in the near future. 

**Please make sure not to include any personal identifying information when requesting assistance. (Examples: postal addresses and bank account information)**

So, bookmark the FAQ, check back often, and rest assured knowing that you'll always be equipped with the latest information to help you along your developer journey. 

<details>
<summary>Click Here For Quick Links 🔗 🔗 </summary>

- <a href="#create-sponsor-profile">Who is eligible to create a Sponsors profile?</a>
- <a href="#sponsor-an-account">Can I sponsor an account?</a>
- <a href="#why-waitlist">Why am I on the waitlist?</a>
- <a href="#havent-received-my-payout">Why haven’t I received my payout?</a>
- <a href="#issue-monthly-payouts">When does GitHub issue monthly payouts?</a>
- <a href="#submit-application-review">How long after I submit my application will it be reviewed?</a>
- <a href="#already-have-github-profile">Why do I have to go through the approval process for my Sponsors profile if I already have a GitHub profile?</a>
- <a href="#complete-listing-description">Do I have to complete my listing description?</a>
- <a href="#switch-invoiced-payments">How do I switch to invoiced payments?</a>
- <a href="#refund-sponsorship">I want to get a refund for my sponsorship, what do I do?</a>
- <a href="#sponsors-account-banned">Why was my Sponsors account banned from the program?</a>
- <a href="#request-account-reinstatement">How do I request an account reinstatement?</a>
- <a href="#couldnt-find-answer">I couldn’t find the answer I was looking for</a>

</details>

<h3 id="create-sponsor-profile">Who is eligible to create a Sponsors profile?</h3>

An individual aged 13+ who contributes to an open source project and lives in a supported region is eligible to become a sponsored developer.

For more information on joining Sponsors, please visit [here](https://docs.github.com/en/sponsors/receiving-sponsorships-through-github-sponsors/setting-up-github-sponsors-for-your-personal-account)

<h3 id="sponsor-an-account">Can I sponsor an account?</h3>

Anyone can sponsor accounts on behalf of their own personal account. Organization owners and billing managers can sponsor accounts on behalf of their organization.

For more information visit [here](https://docs.github.com/en/sponsors/sponsoring-open-source-contributors/sponsoring-an-open-source-contributor-through-github)

<h3 id="why-waitlist">Why am I on the waitlist?</h3>

You don’t live in a supported region for the Sponsors program. We are available wherever Stripe is supported. You have the option of using a fiscal host to receive sponsorships until your region becomes available. 

Please visit [here](https://docs.github.com/en/sponsors/getting-started-with-github-sponsors/about-github-sponsors#supported-regions-for-github-sponsors) for an up to date country and region list. 
Please visit [here](https://docs.github.com/en/sponsors/receiving-sponsorships-through-github-sponsors/using-a-fiscal-host-to-receive-github-sponsors-payouts) for more information on how to become fiscally hosted. 

 
<h3 id="submit-application-review">How long after I submit my application will it be reviewed?</h3>

Applications are generally processed within 6-8 days (excluding holidays & weekends) but may take longer in peak periods such as new country rollouts. 

Please note, our Support team is unable to expedite the review process, so opening a support ticket will not decrease the review time.

<h3 id="#already-have-github-profile"">Why do I have to go through the approval process for my Sponsors profile if I already have a GitHub profile?</h3>

We want to make it as simple as possible for developers anywhere to easily apply and go through the GitHub Sponsors account creation process, while also keeping our commitment to safeguard our partners and ourselves from bad actors who make our mission difficult.

<h3 id="complete-listing-description">Do I have to complete my listing description?</h3>

A complete listing description is not a requirement before submitting your profile for approval but it does speed up the process. A detailed description also gives your funders more insight into your open source contributions.  

<h3 id="refund-sponsorship">I want to get a refund for my sponsorship, what do I do?</h3>

In no event shall GitHub be responsible for providing any support for refunds, nor shall GitHub be liable for payment of any refund. We recommend contacting the maintainer to discuss terms and conditions, if any, for services rendered. 

<h3 id="sponsors-account-banned">Why was my Sponsors account banned from the program?</h3>

Sometimes our system highlights accounts for additional review. Some examples of suspicious behavior include:
fraud, ransom, violations of intellectual property, or other illegal purposes;
misrepresentation or deception related to the reasons you're raising funds or requesting sponsorship

Read more on terms of service violations [here](https://docs.github.com/en/site-policy/github-terms/github-sponsors-additional-terms#6-term-and-termination)

<h3 id="request-account-reinstatement">How do I request an account reinstatement?</h3>

If by any reason, your account was flagged or banned for suspicious activity. Please contact Support and open a support ticket for a review. You may be asked to provide supporting documentation to assist in the review process.

### Why was my sponsorship canceled?

There are several reasons why a sponsor might cancel a sponsorship, including:
- The sponsor no longer has the budget to continue supporting.
- Billing or payment issues.
- The sponsor no longer uses the project.

If you need more information, you can reach out directly to the sponsor. Many sponsors list their emails in their GitHub profiles.

<h3 id="couldnt-find-answer">I couldn’t find the answer I was looking for</h3>

We continuously update these FAQs but if you are unable to find the answer to your question please raise a discussion on the [GitHub Sponsors community](https://github.com/orgs/community/discussions/new?category=sponsors) and the Community team will be happy to assist 🌟

Let's Build Together :octocat:

**Comment by giovannivicentin** [comment]
Thank you for bringing up your questions right here in our GitHub community! It's great to see engaged members seeking clarity on the Sponsors program, and I'm here to shed light on some of the key points:

1. **Creating a Sponsors Profile**: If you're 13 or older, contribute to open source, and are in a Stripe-supported region, you're all set to start your Sponsors journey. This opens up new avenues for supporting and being supported within the open source ecosystem we all value.

2. **Sponsoring Made Simple**: Whether you're an individual eager to support a fellow developer or an organization looking to back meaningful projects, GitHub Sponsors facilitates this beautifully, reinforcing our shared commitment to open source.

3. **Understanding the Waitlist**: If you find yourself waitlisted, it's likely due to regional availability. Meanwhile, consider using a fiscal host to begin receiving sponsorships, ensuring you're ready to hit the ground running once your region is supported.

4. **Navigating Payouts**: New sponsors, note the 60-day waiting period for your first payout, a safeguard for the program's integrity. Subsequent payouts follow a more swift 30-day cycle, aligning with our aim to support your work consistently.

5. **Payout Schedule Details**: Using Stripe Connect? Mark the 22nd on your calendar for monthly payouts, keeping in mind the various factors like processing times that might affect the schedule.

6. **Application Insights**: Our review process, typically spanning 6-8 days, ensures every application is thoroughly vetted, contributing to a trustworthy community.

7. **Profile Approval Process**: Despite having a GitHub profile, the Sponsors profile goes through a separate approval to maintain a secure environment for all developers and sponsors alike.

8. **The Power of Descriptions**: Crafting a compelling listing description, while not mandatory, can significantly enhance your profile's visibility and connection with potential sponsors.

9. **Invoiced Payments for Organizations**: Organizations can streamline their sponsorship contributions through invoiced payments directly from the sponsors dashboard, simplifying financial transactions.

10. **Refund Queries**: For refund-related questions, it's best to engage directly with the project maintainer, as GitHub does not mediate sponsorship refunds.

11. **Addressing Account Bans**: Encountering an account flag? Our Support team is ready to assist, ensuring every case is reviewed with the utmost care.

Don't see your question here or need further assistance? This forum is the perfect place to seek additional guidance or share your invaluable feedback. Together, we're not just building software; we're building a vibrant, supportive open source community. Let's keep the momentum going! 🌟

**Comment by alamenai** [comment]
Hi @DHayes10, Stripe supports my country, but you still did not accept my profile!

**Comment by MayankGhatawal** [comment]
Please check the ticket and support me

**Comment by JorenVsyntra** [comment]
Hey everyone!

**Comment by ahmedakef** [comment]
> If you need to delete a duplicate Stripe account, please contact GitHub support

now, there is no way to contact Github support unless you are a pro user , how can we delete the duplicate stripe account?

**Comment by NaeemAbdullahAkram** [comment]
💸 I want to get a refund for my sponsorship, what do I do?
Sponsorship payments are generally non-refundable, as outlined in the GitHub Sponsors [Terms of Service](https://docs.github.com/en/site-policy/github-terms/github-sponsors-additional-terms#2-sponsorships). However, if you believe you were charged in error (e.g., duplicate charges, technical issues), you can:

Contact GitHub Support through [this form](https://support.github.com/contact).

Include relevant details like:

Sponsor and sponsored account names

The charge amount and date

Why you’re requesting the refund

**Comment by harkerbyte** [comment]
I've submitted request for sponsorship page, it been over a week now and it hasn't been approved nor declined.

Can anyone over there, help take a look at it

**Comment by mohtasimbh** [comment]
Overview

GitHub Sponsors supports developers who contribute to open source.
Available only in regions supported by Stripe.
FAQ helps with common questions and updates.

Eligibility

Age 13+.
Must contribute to open source.
Must live in a supported region.

Sponsorship

Individuals and organizations can sponsor accounts.
Organizations need owners or billing managers.

Waitlist

Happens if your region is not supported.
Option: use a fiscal host until support is available.

Application Process

Review takes 6 to 8 days on average.
No fast-track via support tickets.
Approval exists to prevent fraud and misuse.

Profile Setup

Full description not required but helps approval.
Better descriptions attract more sponsors.

Refund Policy

GitHub does not handle refunds.
Contact the project maintainer directly.

Account Issues

Accounts may be banned for fraud or misuse.
You can request review via support if banned.

Sponsorship Cancellation
Common reasons:

Budget issues
Payment problems
Loss of interest in the project

Help

Use the community forum for questions.
FAQ is updated regularly.Overview

GitHub Sponsors supports developers who contribute to open source.
Available only in regions supported by Stripe.
FAQ helps with common questions and updates.

Eligibility

Age 13+.
Must contribute to open source.
Must live in a supported region.

Sponsorship

Individuals and organizations can sponsor accounts.
Organizations need owners or billing managers.

Waitlist

Happens if your region is not supported.
Option: use a fiscal host until support is available.

Application Process

Review takes 6 to 8 days on average.
No fast-track via support tickets.
Approval exists to prevent fraud and misuse.

Profile Setup

Full description not required but helps approval.
Better descriptions attract more sponsors.

Refund Policy

GitHub does not handle refunds.
Contact the project maintainer directly.

Account Issues

Accounts may be banned for fraud or misuse.
You can request review via support if banned.

Sponsorship Cancellation
Common reasons:

Budget issues
Payment problems
Loss of interest in the project

Help

Use the community forum for questions.
FAQ is updated regularly.
